src/POGOProtos/Map/Pokemon/WildPokemon.proto

```diff
@@ -10,5 +10,5 @@ message WildPokemon {
 	double longitude = 4;
 	string spawn_point_id = 5;
 	.POGOProtos.Data.PokemonData pokemon_data = 7;
-	int32 time_till_hidden_ms = 11;
+	int64 time_till_hidden_ms = 11;
```

Comment:
It's 32-bit but it is allegedly defined in the game as a signed value. Not sure what its initial value what, but it's been decreasing since the game came out. At least when it comes down to 0-900000 it'll be interpreted by the map scanners properly. The big mystery is what it will do after that. Will it continue count down after that and go negative again? Is it a hidden easter egg counting down to 2016-09-06 14:56:22 GMT for some reason? Who knows!